README.md

```diff
@@ -14,7 +14,7 @@ The following example configures a project to use the Smithy Gradle plugin:
 ```kotlin
 plugins {
     java
-    id("software.amazon.smithy").version("0.2.0")
+    id("software.amazon.smithy").version("0.3.0")
```

Comment:
This should be 0.4.0 right?

Reply:
This is the version of the plugin, not the IDL. They aren't intended to be in sync, since feature development on both is independent.